unlock-app/src/components/interface/verification/KeyCard.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  AvatarImage,
+  Fallback as AvatarFallback,
+  Root as Avatar,
+} from '@radix-ui/react-avatar'
+import { Lock } from '~/unlockTypes'
+import dayjs from 'dayjs'
+import relativeTimePlugin from 'dayjs/plugin/relativeTime'
+import { addressMinify } from '~/utils/strings'
+import { useConfig } from '~/utils/withConfig'
+import {
+  RiCloseCircleFill as InvalidIcon,
+  RiCheckboxCircleFill as ValidIcon,
+} from 'react-icons/ri'
+import { ReactNode } from 'react'
+
+dayjs.extend(relativeTimePlugin)
+
+export interface Membership {
+  tokenId: string
+  lock: string
+  owner: string
+  expiration: number
+}
+
+interface Props {
+  timestamp: number
+  membership: Membership
+  lock: Lock
+  keyData: any
+  network: number
+  invalid?: string
+  checkedInAt?: string
+  children: ReactNode
+}
+
+export function KeyCard({
+  membership,
+  lock,
+  timestamp,
+  network,
+  keyData,
+  checkedInAt,
+  invalid,
+  children = null,
+}: Props) {
+  const timeSinceSigned = dayjs().from(timestamp, true)
```

Comment:
I've added dayjs which is a very tiny dependency for formatting and manipulating date and time. 

We should use this well tested library for anything related to date/time instead of our own adhoc functions such as `displayAsTextDuration` etc.